### PR TITLE
feat: add unsend template

### PR DESCRIPTION
**Pull request by DJKnaeckebrot**
Adding Unsend Template

### CODE REVIEW

**Comment by DJKnaeckebrot** [comment]
I just saw that if reformatted the whole files spacing :( I'll try to fix that

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: unsend-prod
+
+services:
+  postgres:
+    image: postgres:16
+    networks:
+      - dokploy-network
+    container_name: unsend-db-prod
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
  unsend-db-prod:
    image: postgres:16
    networks:
      - dokploy-network
```

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    # ports:
+    #   - "5432:5432"
+    volumes:
+      - database:/var/lib/postgresql/data
+
+  redis:
+    image: redis:7
+    networks:
+      - dokploy-network
+    container_name: unsend-redis-prod
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
  unsend-redis-prod:
    image: redis:7
    networks:
      - dokploy-network
```

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    #   - "6379:6379"
+    volumes:
+      - cache:/data
+    command: ["redis-server", "--maxmemory-policy", "noeviction"]
+
+  minio:
+    image: minio/minio
+    networks:
+      - dokploy-network
+    container_name: unsend-storage-prod
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
  unsend-storage-prod:
    image: minio/minio
    networks:
      - dokploy-network
```

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      MINIO_ROOT_USER: unsend
+      MINIO_ROOT_PASSWORD: password
+    entrypoint: sh
+    command: -c 'mkdir -p /data/unsend && minio server /data --console-address ":9001" --address ":9002"'
+
+  unsend:
+    image: unsend/unsend:latest
+    networks:
+      - dokploy-network
+    container_name: unsend
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
  unsend:
    image: unsend/unsend:latest
    networks:
      - dokploy-network
```

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      - AWS_ACCESS_KEY=${AWS_ACCESS_KEY:?err}
+      - AWS_SECRET_KEY=${AWS_SECRET_KEY:?err}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:?err}
+      - GITHUB_ID=${GITHUB_ID:?err}
+      - GITHUB_SECRET=${GITHUB_SECRET:?err}
+      - REDIS_URL=${REDIS_URL:?err}
+      - NEXT_PUBLIC_IS_CLOUD=${NEXT_PUBLIC_IS_CLOUD:-false}
+      - API_RATE_LIMIT=${API_RATE_LIMIT:-1}
+    depends_on:
+      postgres:
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
      unsend-db-prod:
```

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        condition: service_healthy
+      redis:
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]:
```suggestion
      unsend-redis-prod:
```

File: apps/dokploy/templates/unsend/index.ts [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    "POSTGRES_PASSWORD=postgres",
+    "POSTGRES_DB=unsend",
+    "DATABASE_URL=postgresql://postgres:postgres@unsend-db-prod:5432/unsend",
+    "NEXTAUTH_URL=http://localhost:3000",
+    `NEXTAUTH_SECRET=${secretBase}`,
+    "GITHUB_ID=",
+    "GITHUB_SECRET=",
+    "AWS_DEFAULT_REGION=us-east-1",
+    "AWS_SECRET_KEY=",
+    "AWS_ACCESS_KEY=",
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/index.ts]:
```suggestion
    "GITHUB_ID='Fill'",
    "GITHUB_SECRET='Fill'",
    "AWS_DEFAULT_REGION=us-east-1",
    "AWS_SECRET_KEY='Fill'",
    "AWS_ACCESS_KEY='Fill'",
```

**Review by Siumauricio — CHANGES_REQUESTED** [review]

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    volumes:
+      - storage:/data
+    environment:
+      MINIO_ROOT_USER: unsend
+      MINIO_ROOT_PASSWORD: password
+    entrypoint: sh
+    command: -c 'mkdir -p /data/unsend && minio server /data --console-address ":9001" --address ":9002"'
+
+  unsend:
+    image: unsend/unsend:latest
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]: can you use a defined version and not the latest?

File: apps/dokploy/templates/unsend/docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+      - dokploy-network
+    restart: always
+    # ports:
+    #   - "6379:6379"
+    volumes:
+      - cache:/data
+    command: ["redis-server", "--maxmemory-policy", "noeviction"]
+
+  unsend-storage-prod:
+    image: minio/minio
```
Comment (Siumauricio) [inline apps/dokploy/templates/unsend/docker-compose.yml]: can you use a defined version?

**Review by Siumauricio — CHANGES_REQUESTED** [review]

**Review by Siumauricio — APPROVED** [review]
Thanks!

**Pull request merged**